wren-ai-service/docs/config_examples/config.azure.yaml

```
@@ -0,0 +1,149 @@
+
+embedding_model_dim: 1536  # Must match model dimension from embedder
+timeout: 120
+recreate_index: true
+
+# For each pipe line component 
+# Replace llm with Azure deployed LLM model
+# Replace Embeddings with Azure deployed Embedding model
+---
+type: pipeline
+pipes:
+  - name: db_schema_indexing
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)